back/src/data.rs

```diff
@@ -105,6 +105,7 @@ pub struct TabCategory {
     pub tab: String,
     pub category: String,
     pub labels: Vec<String>,
+    pub negative_labels: Option<HashSet<String>>,
```

Comment:
Do we need the Option here? I think an empty hashset should be fine in the case that there are no negative labels.

Reply:
With `Option` the `negative_labels` don't need to be present in each entry instead of copy-pasted empty to each tab-category entry for it to be deserializable, which I've thought to be slightly nicer. But I can change that